sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_content_events_v1/query.sql

```diff
@@ -6,7 +6,8 @@ SELECT
           (
             SELECT AS STRUCT
               jsonPayload.fields.* EXCEPT (device_id, user_id),
-              TO_HEX(SHA256(jsonPayload.fields.user_id)) AS user_id
+              TO_HEX(SHA256(jsonPayload.fields.user_id)) AS user_id,
```

Comment:
@kik-kik do we need to hash the user_id? I know for device_id you have an open question out but not sure I understand the user_id part.

Reply:
this logic has always been here, I only added a comma here. I would not modify this.

Reply:
oops missed that